src/ray/common/ray_config_def.h

```diff
@@ -778,3 +778,8 @@ RAY_CONFIG(bool, raylet_core_dump_exclude_plasma_store, true)
 
 /// Whether to kill idle workers of a terminated job.
 RAY_CONFIG(bool, kill_idle_workers_of_terminated_job, true)
+
+// Instruct the Python default worker to preload the specified imports.
+// This is specified as a comma-separated list.
```

Comment:
Add an example to the docstring,

RAY_preload_python_modules=tensorflow,pytorch